integration-tests/cypress/integration/signup.spec.js

```diff
@@ -68,10 +69,12 @@ describe("Sign up", function() {
     cy.get("#btn-signup-confirm").should("be.disabled");
   });
 
-  it("Singup form can be submitted if you fill in everything correctly.", function() {
+  it("Signup form can be submitted if you fill in everything correctly.", function() {
     let user = testUser();
-    cy.visit("/createAccount");
```

Comment:
Router behavior will redirect away from `/createAccount` if the user is not authenticated.

Reply:
Really? That seems like a bug then... /createAccount should be accessible if the user is not authenticated...

Reply:
Yup, that's the behavior I get on my end at least.
Should I fix the bug as well?

Reply:
Ah I get that behaviour too... yes please fix that bug as well, it makes sense for `/createAccount` to be accessible by a non-authenticated user